src/main/java/com/rtb/projectmanagementtool/user/UserSettingsServlet.java

```diff
@@ -59,30 +59,30 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
     majorsString = majorsString.replaceAll("\\s", "");
 
     // pre checked buttons
-    ArrayList<String> checkedStatus = new ArrayList<>(Arrays.asList("", "", "", "", "", "", ""));
+    String[] checkedStatus = {"", "", "", "", "", "", ""};
     Skills getSkills = user.getUserSkills();
     String skillsAsString = getSkills.name();
 
     // currently not using: look up "switch-case"
     if (skillsAsString.equals("NONE")) {
```

Comment:
nit:  map with the skill name as the key would be more readable (and easier to build) since it'll more clearly show which skills are enabled